package.json

```diff
@@ -1,69 +1,69 @@
 {
```

Comment:
This is not my fault actually, the prettier config in the repo specifies 4 tabWidth